src/renderer/components/+extensions/extensions.tsx

```diff
@@ -79,11 +82,11 @@ export class Extensions extends React.Component {
               Description: <span className="text-secondary">{description}</span>
             </div>
           </div>
-          {!isEnabled && (
-            <Button plain active onClick={() => ext.enable()}>Enable</Button>
+          {!enabled && (
+            <Button plain active onClick={() => extensionsStore.setEnabled(extId, true)}>Enable</Button>
           )}
-          {isEnabled && (
-            <Button accent onClick={() => ext.disable()}>Disable</Button>
+          {enabled && (
+            <Button accent onClick={() => extensionsStore.setEnabled(extId, false)}>Disable</Button>
```

Comment:
I really think that this should either be a ternary or a separate function (I lean towards separate function) since that is what you really want (either or).